package.json

```diff
@@ -1,20 +1,16 @@
 {
   "dependencies": {
-    "@codemirror/autocomplete": "0.18.8",
-    "@codemirror/closebrackets": "0.18.0",
-    "@codemirror/commands": "0.18.3",
-    "@codemirror/comment": "0.18.1",
-    "@codemirror/fold": "0.18.2",
-    "@codemirror/gutter": "0.18.4",
-    "@codemirror/highlight": "0.18.4",
-    "@codemirror/history": "0.18.1",
-    "@codemirror/language": "0.18.2",
-    "@codemirror/lint": "0.18.6",
-    "@codemirror/matchbrackets": "0.18.0",
-    "@codemirror/rectangular-selection": "0.18.1",
-    "@codemirror/search": "0.18.4",
-    "@codemirror/state": "0.18.7",
-    "@codemirror/view": "0.18.19",
+    "@codemirror/autocomplete": "^6.0.2",
+    "@codemirror/commands": "^6.0.0",
+    "@codemirror/language": "^6.1.0",
+    "@codemirror/lint": "^6.0.0",
+    "@codemirror/search": "^6.0.0",
+    "@codemirror/state": "^6.0.1",
+    "@codemirror/view": "^6.0.2",
+    "@lezer/common": "^1.0.0",
+    "@lezer/generator": "^1.0.0",
+    "@lezer/highlight": "^1.0.0",
+    "@lezer/lr": "^1.0.0",
     "@tailwindcss/aspect-ratio": "^0.2.0",
     "@tailwindcss/forms": "^0.2.1",
     "@tailwindcss/typography": "^0.5.0",
```

Comment:
Shall I move the @tailwind dependencies to devDependencies? They not directly required from cljs namespaces.